### PR TITLE
Update upload command docs

### PR DESCRIPTION
**Pull request by adi88d**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5436

### CODE REVIEW

File: demisto_sdk/commands/upload/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ and `DEMISTO_API_KEY` environment variable should contain a valid Cortex XSOAR/X
 **Note:** Cortex XSIAM Base URL should be retrieved from XSIAM instance -> Settings -> Configurations -> API Keys -> `Copy URL` button on the top rigth corner, and not the browser URL.
```
Comment (idovandijk) [inline demisto_sdk/commands/upload/README.md:8]: Please also add that it's the same process for XSOAR 8.X.X (verify first that it's true)

Comment (adi88d) [reply demisto_sdk/commands/upload/README.md:8]: same process

**Comment by adi88d** [comment]
> Please see my comment. Also check whether the -x flag is applicable for XSOAR 8.x.x or only for XSIAM.

we need to use the -x also for XSOAR 8.x.x

File: demisto_sdk/commands/upload/README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ and `DEMISTO_API_KEY` environment variable should contain a valid Cortex XSOAR/X
...
 **Note:** Cortex XSIAM API key should be of a `standard` security level, and have the `Instance Administrator` role.
 
-To use the command on Cortex XSIAM the `XSIAM_AUTH_ID` environment variable should also be set.
+To use the command on Cortex XSIAM or Cortex XSOAR 8.1.x, the `XSIAM_AUTH_ID` environment variable should also be set.
```
Comment (dorschw) [inline demisto_sdk/commands/upload/README.md:11]:
```suggestion
To use the command on Cortex XSIAM or Cortex XSOAR 8.x, the `XSIAM_AUTH_ID` environment variable should also be set.
```

**Review by idovandijk — APPROVED** [review]
Nice

**Pull request merged**